### DISCUSSION TITLE
Install multiple extensions with `config.yml`

### DISCUSSION THREAD

**Discussion by cycotron69** [coder/code-server#6234] [Q&A]
Hi, I'm running the latest Docker container image of code-server on my Linux box.  I mount my own `/.config/config.yml` into the container for setup.  However, I'm not able to figure out how to specify multiple extensions to be installed via the `config.yml`, or maybe it's just not possible?  Below are what I've tried and the results.

---
when I try to specify multiple `install-extension` declarations, ex:

```
install-extension: timonwong.shellcheck
install-extension: mads-hartmann.bash-ide-vscode
install-extension: foxundermoon.shell-format
```

I get the duplicate error:

```
error duplicated mapping key in "/home/coder/.config/code-server/config.yaml" (6:1)

 5 | install-extension: timonwong.she ...
 6 | install-extension: mads-hartmann ...
-----^
 7 | install-extension: foxundermoon. ...
```

---
when I try to put all extension on the same line, ex:

```
install-extension: timonwong.shellcheck mads-hartmann.bash-ide-vscode foxundermoon.shell-format
```

I get the extension not found error:

```
Extension 'timonwong.shellcheck mads-hartmann.bash-ide-vscode foxundermoon.shell-format' not found
```

---
I also tried YAML format:

```
install-extension: 
  - timonwong.shellcheck
  - mads-hartmann.bash-ide-vscode
  - foxundermoon.shell-format
```

and got:

```
Extension 'timonwong.shellcheck,mads-hartmann.bash-ide-vscode,foxundermoon.shell-format' not found.
```

**Comment by code-asher** [comment]
Ah yeah we have an issue for fixing that: https://github.com/coder/code-server/issues/6149
